eth/internal/ethapi/api_test.go

```diff
@@ -0,0 +1,104 @@
+package ethapi_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/polymerdao/monomer/eth"
+	"github.com/polymerdao/monomer/eth/internal/ethapi"
+	"github.com/polymerdao/monomer/testutils"
+	"github.com/stretchr/testify/require"
+)
+
+// Constants for commonly used values
+const (
+	zeroString     = "0x0000000000000000000000000000000000000000000000000000000000000000"
+	someAddressHex = "0xabc"
+)
+
+// generateProofAPI creates a ProofAPI instance for testing
+func generateProofAPI(t *testing.T, blocksNumber int) *eth.ProofAPI {
```

Comment:
I don't think it's necessary to take the number of blocks in as a param since we only need to verify the proof with one block.